core/templates/pages/exploration-editor-page/preview-tab/preview-tab.component.html

```diff
@@ -3,7 +3,7 @@
     <div *ngIf="previewWarning">
       <div class="oppia-preview-state-warning">{{ previewWarning }}</div>
     </div>
-    <oppia-conversation-skin></oppia-conversation-skin>
+    <oppia-conversation-skin *ngIf="voiceoversAreLoaded"></oppia-conversation-skin>
```

Comment:
What happens if voiceovers take too long to load? Should there be a loading indicator in place here similar to the exploration player page?

Reply:
Added.